README.md

```diff
@@ -1,2 +1,64 @@
-# sdk-js
-Evefan sdk to manage events and query information
+# Evefan SDK for JavaScript
+
+Evefan SDK to manage events and query information
+
+## Installation
+
+Install the SDK using npm:
+
+```
+npm install @evefan/sdk-js
+```
+
+## API Reference
+
+### `EvefanSDK(writeKey, workerUrl)`
+
+Creates a new instance of the Evefan SDK.
+
+- `writeKey`: Your Evefan write key
+- `workerUrl`: URL of the Evefan worker
+
+### `evefan.query(sqlQuery)`
+
+Executes a SQL query against your Evefan data.
+
+- `sqlQuery`: SQL query string
+
+Returns a Promise that resolves with the query results.
+
+## Code Snippet for Node.js
+
+Here's a standalone code snippet you can use as a starting point:
+
+```
+const EvefanSDK = require("@evefan/sdk-js").default;
```

Comment:
Can we call the exported object just 'Evefan' and not EvefanSDK?

Reply:
Updated in the last commit.